src/main/java/net/arcaniax/buildersutilities/menus/SecretBlockMenuProvider.java

```diff
@@ -82,6 +79,9 @@ public void init(Player player, InventoryContents contents) {
                 5,
                 ClickableItem.of(STRUCTURE_BLOCK, inventoryClickEvent -> player.getInventory().addItem(STRUCTURE_BLOCK))
         );
+        if(LIGHT != null) {
+            contents.set(0, 6, ClickableItem.of(LIGHT, inventoryClickEvent -> player.getInventory().addItem(LIGHT)));
+        }
```

Comment:
Style doesn't match project style rules, please adjust.

Reply:
May be OK now.
CONTRIBUTING.md at root of project is not available.
There are a "java" cs fixer ?

Reply:
There's an `.editorconfig` file available in the root directory, IntelliJ will use if you have the editor config plugin.

Reply:
Yep, I have IDEA Community Edition, but i didnot have any high-light on this

Reply:
Indeed, this should've been resolved by https://github.com/Arcaniax-Development/Builders-Utilities/commit/be00521b9e612459152dacf9145e1c7d161476db

Reply:
PR rebase with master, no error with `./gradlew clean build`